.travis.yml

```diff
@@ -0,0 +1,22 @@
+---
+language: node_js
+node_js:
+  - "7"
+cache:
+  directories:
+  - frontend/node_modules/
+install:
+  - cd frontend/
+  - npm install
+script:
+  - pwd
+  - CI=false npm run build
```

Comment:
What does `CI=false` result in?

Reply:
It allows the build to pass when there's warnings.

    Line 7:  'setUserName' is assigned a value but never used  no-unused-vars
    Line 8:  'setUserID' is assigned a value but never used    no-unused-vars